### PR TITLE
draft 2020-09-29 blog

### PR DESCRIPTION
**Pull request by mpadge**
* [x] I have read the Content Guidelines.
* [x] I have read the Technical Guidelines.
* [x] I used or followed the R Markdown or Markdown template.
* [x] I have followed the Style Guide.
* [x] I created or updated my author metadata with correct folder name.
* [x] I have added relevant tags after browsing existing tags.
* [x] I ran `roblog::ro_lint_md()` on index.md (optional).
* [x] I ran `roblog::ro_check_urls()` on index.md (optional).
* [x] I ran a spell-check on index.md.
* [x] I have added the tags - Software Peer Review, my-packagename, R.
* [x] I have added the package-version YAML tag.
* [x] I have added acknowledgement of the reviewers' work.
* [x] I have added a link to the software peer review thread.
* [x] I ran a spell-check on index.md.

### CODE REVIEW

**Review by steffilazerte — COMMENTED** [review]
Hi Mark, forgive me if I missed how to interact with this post on HackMD!

First off, mapscanner is soooo cool! I have never before felt the need to document trees in my neighbourhood, but I really want to now. I'm also thinking about how this might be used for my Christmas Bird Count route :grin:

On the whole, the post is comprehensive and well written. I have a couple of suggestions (16) mostly regarding small tweaks and stylistic details. I do have one 'larger' suggestion regarding either the order of the use cases, or the content of the 1st use case. These are just suggestions, however, and please feel free to adopt, adjust, or ignore any and all!

I'm also in a bit of a funny situation right now so haven't checked some of the more persnickety details that I usually check locally (links, etc.). My computer died and I'm working off my backup laptop. The parts for my new computer are set to arrive today/tomorrow so I should be up and running by Thursday to do the last checks. I thought I'd best post my initial review now, though, to give you time to make any changes you'd like to make. Any persnickety details I figure out on Thursday, will be small.

If you have time to address these comments by Monday noon (Central time, North America), we're good to go for a Tuesday publication!

* [X] post follows Content Guidelines
* [ ] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [x] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
    - **Your image doesn't pop up, but I'm not sure why, I'll look into this...**
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [x] I ran `roblog::ro_lint_md()` on index.md (**Will do later this week**)
* [x] I ran `roblog::ro_check_urls()` on index.md (**Will do later this week**)
* [x] I ran a spell-check on index.md (**Will do later this week**)
* [X] YAML subject tags are ok
* [X] YAML package-version included
* [X] YAML subject tags - software peer review, packagename, R; add "community" for post by non-staff non-editor
* [X] acknowledges and links to reviewers
* [X] links to peer review thread

**Comment by steffilazerte** [comment]
@maelle @stefaniebutland, the twitter card validator doesn't show the image @mpadge put in the YAML. I can't see why not and since this is the first post I've dealt with that actually supplied an image (:scream_cat:), I have no other examples to drawn on. Feel free to point me to a previous post that does have a twittercard with image!

**Comment by stefaniebutland** [comment]
aha! slashes needed in date e.g. `twitterImg: blog/2020/04/17/news-apr2020/2months2mins_hex_apr2020_wh_300.png`

The twitter card preview works for this post: 
live: https://ropensci.org/blog/2020/04/17/news-apr2020/
source: https://github.com/ropensci/roweb2/tree/master/content/blog/2020-04-17-news-apr2020

**Comment by stefaniebutland** [comment]
@mpadge Can you add couple of words to title to help people see what the pkg does?

**Comment by mpadge** [comment]
@stefaniebutland:

> @mpadge Can you add couple of words to title to help people see what the pkg does?

```
title: Rectifying hand-drawn marks on maps with the mapscanner package
```
Better?

**Comment by mpadge** [comment]
The [above commit](https://github.com/ropensci/roweb2/pull/729/commits/78778d558518c6ff6eac4a4be357f320392be860) should fix the twitter links. I'll likely change the files in response to the excellent suggestions of @steffilazerte, hopefully safe in the knowledge that they'll work from here on.

File: content/blog/2020-09-29-mapscanner/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+portable screen-based devices, have transformed the practice of social
+sciences. I would happily wager that almost everyone would admit that the last
+survey or questionnaire they completed was done with some kind of computer
+hardware, rather than with pen and paper. And yet conducting social research
+with computer hardware remains impossible for very many scientists, who are
+instead restricted to what many would generally refer to as "less"
+technologically sophisticated solutions like pens and paper.
+
+One technological development that has particularly impacted both life in
+general, and the practice of social sciences, has been interactive mapping
```
[the remark this answers is not in the document]
Comment (mpadge) [reply content/blog/2020-09-29-mapscanner/index.Rmd:62]: i added,

> -- think how easy it is to use a smartphone to find somewhere to eat in an unfamiliar city.

Good? Or should i explain a bit further?

**Comment by mpadge** [comment]
MP note to self: TODO (and no merging until these are all checked!!)

- [ ] Put point-based tree stuff first, and polygon-based technicalities second, as per @steffilazerte's suggestion
- [ ] Change images (and likely some text) for polygon analyses to focus on the primary functionality of the package, rather than the secondary functionality of aggregating polygons, as per @steffilazerte's suggestion.

File: content/blog/2020-09-29-mapscanner/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+
+This relationship was artificially contrived to approximately reflect the
+results of [Montello and
+others (2003)](http://geog.ucsb.edu/~montello/pubs/downtown.pdf), who depicted
+polygons approximately converging on to a consensus definition of the downtown
+area. The authors of that paper were, however, able to do little beyond
+providing graphical illustrations of overlapping polygons, preventing any
+general conclusions about the perceived size or location of the downtown area.
+The above graph reflects the power of
+[mapscanner](https://github.com/ropensci/mapscanner)'s ability to [aggreate
```
Comment (steffilazerte) [inline content/blog/2020-09-29-mapscanner/index.Rmd:206]: Minor typo
```suggestion
[mapscanner](https://github.com/ropensci/mapscanner)'s ability to [aggregate
```

File: content/blog/2020-09-29-mapscanner/index.Rmd (hunk 1 above)
[the remark this answers is not in the document]
Comment (steffilazerte) [reply content/blog/2020-09-29-mapscanner/index.Rmd:62]: Nope that's great, just wanted something a bit more recognizable :smile_cat:

File: content/blog/2020-09-29-mapscanner/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+any marks made by hand upon the printed maps, and a single computer capable of
+running R. The package is likely to be of general utility for spatial analyses
+in general, in domains such as geography, ethnography, anthropology,
+archaeology, ecology, botany, political science, and many more, with potential
+use cases as diverse as the variety of hand-drawn marks on maps. 
+
+The following section provides a very brief overview of functionality, followed
+by considering a few use cases to which the package may be applied.
+
+### Package Functionality
```
Comment (steffilazerte) [inline content/blog/2020-09-29-mapscanner/index.Rmd:95]: Non-title case here
```suggestion
### Package functionality 
```

File: content/blog/2020-09-29-mapscanner/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,300 @@
+#
+---
+slug: "mapscanner"
+title: Rectifying hand-drawn marks on maps with the mapscanner package
```
Comment (steffilazerte) [inline content/blog/2020-09-29-mapscanner/index.Rmd:4]: Should be title case :grin:
```suggestion
title: Rectifying Hand-Drawn Marks on Maps With the mapscanner Package
```

**Review by steffilazerte — COMMENTED** [review]
Hi @mpadge, I got my computer up and running and finished the last couple of items. Only a couple of stylistic tweaks to the title and headings, and one typo!

**Comment by mpadge** [comment]
I've checked my items above, completely restructured the whole piece, and am quite satisfied with how it is now. I even managed to add descriptions and links to one of my long-time fave bits of not-quite-science, although opted not to include the official title of [This isn't fucking Dalston](https://sites.google.com/view/tifd/home), much as i was tempted ... :smirk: Let me know what you think, and I'll ensure immediate turn around of any remaining tweaks in time for scheduled publication on the morrow. Oh, and yes, @stefaniebutland the piece is also an intentional acknowledgement that Vancouver rocks!!

**Pull request closed without merging**